cloudformation/membership-attribute-service.yaml

```diff
@@ -71,8 +71,8 @@ Mappings:
     PROD:
       NotificationAlarmPeriod: 1200
       InstanceName: PROD:membership-attribute-service
-      DynamoDBTable: arn:aws:dynamodb:*:*:table/MembershipAttributes-PROD
-      DynamoDBTableTestUsers: arn:aws:dynamodb:*:*:table/MembershipAttributes-UAT
+      DynamoDBTable: "Memb-Attributes-Tables-PROD-SupporterAttributesFallback"
+      DynamoDBTableTestUsers: "Memb-Attributes-Tables-UAT-SupporterAttributesFallback"
```

Comment:
these new table names assume that the tables cloudformation stacks were created with the name Memb-Attributes-Tables-[TOUCHPOINT_BACKEND]